### ISSUE TITLE
Merge Shop wrong More Details position, also one bugs with it.

### ISSUE THREAD

**Issue by DragoNext** [DragoNext/AqwDoIhave#10]
Need to fix position so it is before  the text not after it (It Makes it look worse)
From:
![image](https://user-images.githubusercontent.com/49029552/204102429-e21e58b6-872e-4d2e-a63b-b247eb0f15cf.png)
To:
![image](https://user-images.githubusercontent.com/49029552/204102466-ff72dd06-0d15-454e-b412-aa2a54ef6c40.png)

Icon that is from Bank Or Inventory Should Be on right as i consider it as Tag not As More Details.

Fix:
Doesn't properly display shop name.
![image](https://user-images.githubusercontent.com/49029552/204102526-057e41a0-fbb2-4266-82b8-573ff3f1e230.png)

Performance of it:
Is it slow as it is gets data in real time, in future it will be improved as it will use scraped data.

**Comment by Quannh1511** [comment]
Good idea, I am very important about performance, hope you have a solution to improve the logic of the code. It should be threaded and look easy to read.

**Comment by Quannh1511** [comment]
I usually don't make a function too long, but rather break it down into the features it does, making it easier to maintain and read as well.

**Comment by DragoNext** [comment]
Fixed positions of Icon, now will work on code to use scraped data to index items details, instead of making request to aqwiki and scrapping it in real time, it should improve performance tremendously.
![image](https://user-images.githubusercontent.com/49029552/204818752-22fc675b-e264-4c95-92a3-e26a28535ef1.png)

**Comment by Quannh1511** [comment]
Here is my illustrative example. If possible, you can make the filter display as text or icon.
![image](https://user-images.githubusercontent.com/107485615/205048900-5338fb55-c9cd-4206-9528-0a3b0699ab48.png)

**Comment by Quannh1511** [comment]
Hi, I'll be back soon. I need your info from discord. It will be easier for us to talk.

**Comment by Quannh1511** [comment]
Hello, I'm back. I need your information from Discord. It would be easier for us to talk there.
